fmt/xi.c

```diff
@@ -107,9 +107,7 @@ int fmt_xi_read_info(dmoz_file_t *file, const uint8_t *data, size_t length)
 		return 0;
 
 	file->description = "FastTracker Instrument";
-	file->title = mem_alloc(24);
```

Comment:
Rhetorical question: why on earth was this allocating 24 bytes in the first place?